platform/security/idp/security-idp-server/src/main/grunt/tasks/bower.js

```diff
@@ -0,0 +1,33 @@
+module.exports = function(grunt) {
```

Comment:
There isn't a better way just yet. We've talked about some ideas, but we don't have anything concrete yet.